flash.sh

```diff
@@ -0,0 +1,298 @@
+ADB=${ADB:-adb}
```

Comment:
I'm not really sure we should depend on ADB for this device, since it's not really accessible by default